.github/workflows/validate_tutorials.yaml

```diff
@@ -184,6 +184,60 @@ jobs:
       #     pushd tutorials/observability
       #     make validate
       #     popd
+      - name: Validate workflow/csharp/child-workflows
```

Comment:
Lets keep these all commented out for now and then comment them in once we add additional logic into the mechanical markdown to check for the actual running of the workflow.